images/common/utils.sh

```
@@ -236,18 +236,25 @@ function openvpn_config_checksum {
 }
 
 function openvpn_config_download {
-	curl --silent --retry 10 --retry-delay 5 --retry-max-time 300\
-		--insecure --output vpn.tar.gz \
+	curl --silent --retry 10 --retry-delay 5 --retry-max-time 300 --insecure --output vpn.tar.gz \
 		${API_INTERNAL}/controller/vpn/download-config/$UUID/?key=$KEY
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)